composer.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "nextcloud/limit_login_to_ip",
 	"description": "Lint config for nextcloud/limit_login_to_ip",
-	"license": "MIT",
```

Comment:
Not sure about simply changing the license?

Reply:
You are right.
I believe I saw AGPL v3 somewhere so I wanted to make it consistent, but I don't find it again.
In any case, it should not be in this pull request :+1: